spk/syncthing-inotify/Makefile

```diff
@@ -0,0 +1,37 @@
+SPK_NAME = syncthing-inotify
+SPK_VERS = 0.8.2
+SPK_REV = 1
+SPK_ICON = src/${SPK_NAME}.png
+BETA = 1
+
+UNSUPPORTED_ARCHS = $(PPC_ARCHES)
+
+DEPENDS = cross/busybox cross/$(SPK_NAME)
```

Comment:
Can you use Syncthing's busybox binary instead? That means you should put `/usr/local/syncthing/bin` on $PATH of inotify's dsm-control.sh, but that cleans up this package a bit (drop `cross/busybox` here and drop L29-30)